### DISCUSSION TITLE
NextAuth Google Signin Issue in production when deployed in Vercel

### DISCUSSION THREAD

**Discussion by mrjaysonwong** [vercel/community#2628] [Help]
### Summary

Hi, I would like to ask if the previous status DNS timeouts for Google Provider is the reason why my NextAuth GoogleProvider Signin does not work, it was working a few weeks ago, I have version  "next": "^12.3.1",  "next-auth": "^4.18.5", all secret/key/id is correct as well as in Google Console, my GitHub signin' in production is working, I have not touched the codebase for few weeks.

### Steps to Reproduce

got this callbackURL issue
https://touring.vercel.app/login?callbackUrl=https%3A%2F%2Ftouring.vercel.app&error=OAuthCallback,
first, it has status code 200, then pending, and after reload status code 304.

**Comment by z4nr34l** [comment]
Could you bring us more details?
I consider as helpful those informations:
1. Repository Access;
2. Vercel build logs (from deployment with this issue);
3. Runtime logs from problematic request;

The more - the better for receiving quick help.

You can also try to reproduce this problem on free [CodeSandbox](https://codesandbox.io/) and post URL here when it's done.
